src/System Application/App/User Settings/src/UserSettingsImpl.Codeunit.al

```diff
@@ -368,6 +370,40 @@ codeunit 9175 "User Settings Impl."
         ApplicationUserSettings.Modify();
     end;
 
+    procedure LegacyActionBarEnabled(UserSecurityId: Guid): Boolean
```

Comment:
The variable name UserSecurityId is the same as the global function name UserSecurityId(), this could be confusing.

Reply:
We should probably have a rule to prevent that, just like we have for not naming variables the same as functions.

Reply:
Standarized to the other methods, `UserSecurityID` instead of `UserSecurityId`. Not going in to change all methods to use `UID`.